Security/src/EOMT.ps1

```diff
@@ -600,6 +603,31 @@ function Get-ServerVulnStatus {
     return $false
 }
 
+function Get-PatchMessage {
+    $Version = (Get-ItemProperty 'HKLM:\SOFTWARE\Microsoft\ExchangeServer\v15\Setup\').OwaVersion
+    $Message = "Exchange On-premises Mitigation Tool has mitigated against known attacks. For long-term protection, please use Microsoft Update to install the latest Security Update for Exchange Server (KB5000871)."
+
+    if ($Version -like "15.2.*") {
+        return $Message
+    }
+    elseif ($Version -like "15.1.*") {
+        $Message += "`nIf you don't see this security udpate, please upgrade to Exchange 2016 Cumulative Update 20 via: "
+        $Message += $exchange2016CU20Downloadlink
+
+        return $Message
+    }
+    elseif ($Version -like "15.0.*") {
+        $Message += "`nIf you don't see this security udpate, please upgrade to Exchange 2013 Cumulative Update 23 via: "
+        $Message += $exchange2013CU23Downloadlink
+        $Message += "`nYou will need to download the latest security update from link "
```

Comment:
```PowerShell
"After applying the cumulative update, you will also need to install the latest security update: $exchange2013CU23SecurityUpdateDownloadLink"
```

Reply:
Should I use 2 lines or combine into single line?

        $Message += "`nIf you don't see this security update, please upgrade to Exchange 2013 Cumulative Update 23 via: $exchange2013CU23Downloadlink"
        $Message += "`nAfter applying the cumulative update, you will also need to install the latest security update: $exchange2013CU23SecurityUpdateDownloadLink"